config/routes.rb

```diff
@@ -55,6 +55,8 @@
 
   get 'search' => 'search#index'
   post 'create_multiple_evidences' => 'evidence#create_multiple'
+  get 'trash' => 'revisions#trash'
```

Comment:
We could do, but it would mean changing the URL `trash` to `revisions/trash` if we wrap this line a `resources` block.

The line directly below this one could be replaced with the functionally-identical:

    resources :revisions, only: [] do
      member { post :recover }
    end

... if you think that's better (it *is* more "Railsy")